tools/pot/docs/SimplifiedMode.md

```diff
@@ -8,7 +8,26 @@ Simplified mode is designed to make data preparation for the model optimization
 
 ## Usage
 
-To use the Simplified mode, prepare the data and place it in a separate folder. No other files should be present in this folder. There are two options to run POT in the Simplified mode:
+To use the Simplified mode, prepare the data and place it in a separate folder. No other files should be present in this folder.
+
+To apply optimization when there is only a model and no data is available. It is possible to generate a synthetic dataset using Dataset Management Framework (Datumaro) available on [GitHub](https://github.com/openvinotoolkit/datumaro). Currently, data generation is available only for Computer Vision models, it can take time in some cases.
+
+Install Datumaro:
+
+``` bash
+pip install datumaro>=0.4.0.1
+```
+Create a synthetic dataset with elements of the specified type and shape, and saves it in the provided directory.
```

Comment:
I would say a couple of words about how to control the number of images and resolution. I think providing certain example would be beneficial here, e.g.:
```
datum generate  -o ./dataset -k 300 --shape (224,224,3)
```